packages/vm/tests/api/runCode.spec.ts

```diff
@@ -21,7 +21,7 @@ const testCases = [
 tape('VM.runCode: initial program counter', async (t) => {
   const vm = await VM.create()
 
-  for (const [i, testData] of testCases.entries()) {
+  for (const [, testData] of testCases.entries()) {
```

Comment:
if you don't need the index, i think you can just do `for (const testData of testCases) {` since it is already an array. 

in general, for the future, i prefer to use underscore prefix to describe a variable as unused (e.g. here `_i`) rather than blank, i find it easier to follow.

Reply:
Re-added the i since I reverted to the old error message with the index